ex03.ex

```diff
@@ -101,7 +116,11 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([], []), do: true
```

Comment:
## -2

This code works, but not for the reason you think.

~~~ elixir
def list_equal([h | t1], [h | t2]), do:
	list_equal([t1], [t2])
end
~~~

Say you call `list_equal([1, 2, 3], [1, 2, 3])`, the recursive call would call `list_equal([[2, 3]], [[2, 3]])`

This is two single element lists, so the call will match [2,3] and [2,3].